CONTRIBUTING.md

```diff
@@ -142,21 +142,24 @@ tracing) enabled.
 The setup for a local rustc works as follows:
 ```sh
 # Clone the rust-lang/rust repo.
-git clone https://github.com/rust-lang/rust/ rustc
+git clone https://github.com/rust-lang/rust rustc
 cd rustc
-cp config.toml.example config.toml
-# Now edit `config.toml` and set `debug-assertions = true`.
+# Create a config.toml with defaults for working on miri
+./x.py setup compiler
 
-# Build a stage 2 rustc.
+# Build stage2 rustc using the stage1 compiler.
```

Comment:
I don't think we should say "build stage 2 rustc" when we do `--stage 1`, that is just really confusing.

Also there is some open discussion about this in https://github.com/rust-lang/miri/pull/1612, so now things are somewhat messy as it is unclear where to give feedback for these comments.^^

Reply:
Sure, that suggestion looks fine to me, as long as it's clear that rustc is being built twice, not once.